README.md

```diff
@@ -93,10 +93,16 @@ Re-run `. .venv/bin/activate` in each shell.
     ```bash
     aws-azure-login --profile=<AWS-PROFILE-NAME>
     ```
+1. Environment variables
+* **DEPLOY_ENV:** set deployment environment. Recommended values: prod, nonprod, ci, dev or any
+    string without spaces
+* **DATALAKE_USE_EXISTING_VPC:** determine if networking stack will use existing VPC in target AWS
```

Comment:
We should document the default here.

Reply:
It is documented now and `DATALAKE_USE_EXISTING_VPC` default is `false`